riptide-spring-boot-starter/src/main/java/org/zalando/riptide/spring/RiptideClientTest.java

```diff
@@ -0,0 +1,36 @@
+package org.zalando.riptide.spring;
+
+import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.client.AutoConfigureMockRestServiceServer;
+import org.springframework.boot.test.autoconfigure.web.client.RestClientTest;
+import org.springframework.core.annotation.AliasFor;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@RestClientTest
+@AutoConfigureMockRestServiceServer(enabled = false) // will be registered per client
```

Comment:
Comment is out-dated. Can you describe here why we need this?

Reply:
Ups will do